test/test_luaunit.lua

```diff
@@ -138,6 +138,23 @@ TestLuaUnitUtilities = { __class__ = 'TestLuaUnitUtilities' }
         lu.assertEquals( v, '33' )
     end
 
+    function TestLuaUnitUtilities:test_randomizeTable()
+        local t, tref, n, i
+        t, tref, n, i = {}, {}, 20, 0
+        while i < n do
+            table.insert(t, i)
+            table.insert(tref, i)
+            i = i + 1
+        end
```

Comment:
We could even avoid the `table.insert` easily by using [1, n] instead of [0, n-1]:
```lua
        local t, tref, n = {}, {}, 20
        for i = 1, n do
            t[i], tref[i] = i, i
        end
```